lib/screen/book_editor.dart

```diff
@@ -67,6 +74,38 @@ class _BookEditorState extends State<BookEditor> {
       ),
       ),
       ),
+    floatingActionButtonLocation: FloatingActionButtonLocation.endFloat,
+    floatingActionButton: FloatingActionButton.extended(
+    backgroundColor: Theme.of(context).accentColor,
+    onPressed: () => _onSave(uid),
```

Comment:
Looks like _onSave is a member funciton of this class right? Then we don't need to pass it the uid because the uid is also a member property of the class. Isn't it right?

Reply:
Yes it is right, I will update it and see the result

Reply:
I just looked at the code and I am seeing that the uid is just define in the build function, so it is not a class member. In that case I need to pass it to the function